examples/research_projects/wav2vec2/README.md

```diff
@@ -0,0 +1,8 @@
+## Fine-tuning Wav2Vec2
+
+The `run_training.py` script allows one to finetune pretrained Wav2Vec2 models that can be found [here](https://huggingface.co/models?search=facebook/wav2vec2).
```

Comment:
We usually name the run script with the ask, so `run_wav2vec`?

Reply:
I would prefer `run_asr`, we name scripts for tasks, `wav2vec` is not a task